scripts/lab_state_refresh.sh

```diff
@@ -1,6 +1,14 @@
 tekton_project=tekton-pipelines
 operators_project=operators
 
+
+updateOperators() {
+    oc patch deploy/halkyon-operator \
+       --patch '{"spec":{"template":{"spec":{"containers":[{"name":"halkyon-operator", "image":"quay.io/halkyonio/operator:v0.1.8"}]}}}}' \
```

Comment:
Why do we have to patch the current lab ? What should we do tomorrow, in the future when new operator releases will become available ? How do you install this lab on Ravello ? @jbride